Sources/protoc-gen-grpc-swift/Generator-Names.swift

```diff
@@ -140,6 +139,18 @@ extension Generator {
     return self.sanitize(fieldName: fnName)
   }
 
+  internal var methodMakeFunctionCallWrapperName: String {
+    return "make\(methodComposableName)Call"
+  }
+
+  internal var methodComposableName: String {
+    var name = method.name
+    if !options.keepMethodCasing {
+      name = name.prefix(1).uppercased() + name.dropFirst()
```

Comment:
We should be using the camel caser here to make it upper camel-case

Reply:
You've missed the whole point. We can't use camel caser here because that's the thing causing the issue.
For `rpc GRPCUnary(google.protobuf.Empty) returns (google.protobuf.Empty) {}` it produces `makeGrpcunaryCall`. None of the naming in grpc-swift uses the `NamingUtils` except in this one place, which is why it's inconsistent with the rest of the generated code.
`methodFunctionName` doesn't use it either. The PR follows the same logic as in `methodFunctionName` so that the generator produces a unified code.